docs/README.md

```diff
@@ -1856,6 +1856,38 @@ mutation {
 }
 ```
 
+It is also possible to use a GraphQL client to subscribe to changes in the application. However, it might be necessary to include an access token, provided when the user signs-in, in the request if the operation requires authorization. This token should be added to the GraphQL client as an operation option through a middleware. An example can be seen below:
```

Comment:
@alvaroloes @itrion I added some more information about how to include the auth token in subscriptions

Reply:
Thanks! In any case, I am writing and rephrasing the GraphQL documentation and I will include this new way of passing the token ;-)